packages/dds/tree/src/test/snapshots/summary/Compressed/v1/optional-field-scenarios-final.json

```diff
@@ -330,13 +330,13 @@
                                             "change": {
                                               "m": [
                                                 [
-                                                  1,
+                                                  3,
```

Comment:
Did you manage to figure out why this has changed? It's not inherently an issue that it has, I just want to make sure we understand why.